screen.css

```diff
@@ -31,6 +31,156 @@ body {
   line-height: 20px;
 }
 
+#Departflights ul {
+  display: inline-block;
+}
+
+#ReturnFlights ul{
+  display: inline-block;
+}
```

Comment:
the order that css files ought to be in for the most readability is general, full body styles at the top, and then styles slowly increasing in specificity as you progress downwards through the file. so the first tag on the styles page should be `html {}` and then something like `returnflights ul {} ` would be nearer the bottom, in its own particular section. so i would suggest moving this. if you want me to show you where exactly, i can. 
_however_, this whole portion of your css file i'd actually recommend deleting because the bottom half is my code and the top seems like a straight duplicate. see my other comment below this one